src/main/docs/guide/azureCosmos/azureCosmosCriteriaSpecifications/azureCosmosCriteriaExecuteQuery.adoc

```diff
@@ -62,7 +62,7 @@ Then you can combine them for `find` or `count` queries:
 
 snippet::example.PersonRepositorySpec[project-base="doc-examples/azure-cosmos-example",source="test" tags="find",indent="0"]
 
-Specific criteria supported only by Micronaut Azure Cosmos Data is `ARRAY_CONTAINS` and for a class having an array of strings named `tags` it can be used either via custom repository method like this:
+Specific criteria supported Micronaut Azure Cosmos Data is `ARRAY_CONTAINS` and for a class having an array of strings named `tags` it can be used either via custom repository method like this:
```

Comment:
Please also add it to https://micronaut-projects.github.io/micronaut-data/latest/guide/#criteria

Reply:
Didn't find good place where to put it but will add it somewhere.

Reply:
Added some docs

Reply:
It should be in that table with all possible criteria, with a note that it's only for some DBs

Reply:
Added